<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tedsDialog</class>
 <widget class="QDialog" name="tedsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TEDS</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Gegevens in Opnemer</string>
       </property>
       <widget class="QLineEdit" name="refSensStored">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>21</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>9.9999; </string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>23</y>
          <width>50</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ref. Sens.</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>82</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Kal. jaar</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="calYearStored">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="calMonthStored">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="readSensor">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Lees Opnemer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>52</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>1 - 12</string>
        </property>
        <property name="text">
         <string>Kal. maand</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Nieuwe Gegevens</string>
       </property>
       <widget class="QLineEdit" name="calMonthNew">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>82</y>
          <width>50</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Kal. jaar</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="calYearNew">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="refSensNew">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>21</y>
          <width>93</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>9.9999; </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>52</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>1 - 12</string>
        </property>
        <property name="text">
         <string>Kal. maand</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>23</y>
          <width>50</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ref. Sens.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="editButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="writeSensor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Schrijf naar Opnemer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Sluit</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>refSensNew</tabstop>
  <tabstop>calMonthNew</tabstop>
  <tabstop>calYearNew</tabstop>
  <tabstop>writeSensor</tabstop>
  <tabstop>calYearStored</tabstop>
  <tabstop>calMonthStored</tabstop>
  <tabstop>refSensStored</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>tedsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
